src/api.v2/controllers/cellLevelController.js

```diff
@@ -0,0 +1,48 @@
+const { v4: uuidv4 } = require('uuid');
+const _ = require('lodash');
+const bucketNames = require('../../config/bucketNames');
+const sqlClient = require('../../sql/sqlClient');
+const CellLevel = require('../model/CellLevel');
+const CellLevelToExperiment = require('../model/CellLevelToExperiment');
+const { getFileUploadUrls } = require('../helpers/s3/signedUrl');
+const OK = require('../../utils/responses/OK');
+
+const uploadCellLevelMetadata = async (req, res) => {
+  const { experimentId } = req.params;
+  const { name, size } = req.body;
+
+  const cellLevelKey = uuidv4();
+  const bucketName = bucketNames.CELL_METADATA;
+  const newCellLevelFile = {
+    id: cellLevelKey,
+    name,
+    upload_status: 'uploading',
+  };
+  const cellLevelToExperimentMap = {
+    experiment_id: experimentId,
+    cell_metadata_file_id: cellLevelKey,
+  };
+
+  let uploadUrlParams;
+  await sqlClient.get().transaction(async (trx) => {
+    await new CellLevel(trx).create(newCellLevelFile);
+    await new CellLevelToExperiment(trx).create(cellLevelToExperimentMap);
+    uploadUrlParams = await getFileUploadUrls(cellLevelKey, {}, size, bucketName);
+    uploadUrlParams = { ...uploadUrlParams, fileId: cellLevelKey };
+  });
+
+  res.json({ data: uploadUrlParams });
+};
+
+
+const updateCellLevelMetadata = async (req, res) => {
+  const { params: { experimentId }, body } = req;
+  const snakeCasedKeysToPatch = _.mapKeys(body, (_value, key) => _.snakeCase(key));
+  const { cellMetadataFileId } = await new CellLevelToExperiment().find({ experiment_id: experimentId }).first();
+  await new CellLevel().updateById(cellMetadataFileId, snakeCasedKeysToPatch);
```

Comment:
This call won't work, also we are allowing users to decide what to patch, and that's not something we want to do.

Reply:
This isn't part of the ticket anyways, so maybe you could just leave this function empty

Reply:
I'm using this to update the status of the upload. What's the problem here? Why would it not work?

Reply:
Because it allows you to update anything you want, not only the status.

I would suggest trying to mimic the patch of sample files `const patchFile = async (req, res) => {`